nightlies/cron_wrapper

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# /nightlies/cron_wrapper.sh
+
+if [ -z $1 ]; then
+  echo "need argument"
+  exit 1
+fi
+
+BASENAME=`basename $0`
+LOG="/var/tmp/$BASENAME.$$.txt"
+
+"$@" &>"$LOG"
+
+rc=$?
+
+#if there is an error, eamil to MAILTO in crontab
+if [ $rc != 0 ]; then
+        echo "return code $rc"
+        cat "$LOG"
+fi
+
+#trap "rm -f $TMP" 0
+rm -f $LOG
```

Comment:
If we logged, we'd set up logrotate, of course.   I dunno; I'm nervous about losing success logs altogether.

Also, please make the link Fixes: <url-to-tracker>; sorry, I should have specified more carefully, but that's what we've been doing for the last few months.